Update LICENSE.md (#1838)
Nemocas/Nemo.jl: LICENSE.md

Various parts of Nemo are licensed differently. 

Individual parts of Nemo may be distributed according to more permissive terms as laid out below:

> * Parts of the file src/flint/fmpz.jl is licensed with the MIT license (see ZZ.jl for precise conditions of the license).

Reviewer: ```suggestion
* Parts of the file src/flint/fmpz.jl is licensed with the MIT license (see the top of that file for precise conditions of the license).
```
still the old name

> * Parts of the file src/flint/fmpz.jl is licensed with the MIT license (see the top of that file for precise conditions of the license).

Reviewer 2: ```suggestion
* Parts of the file src/flint/fmpz.jl are licensed under the MIT license (see the top of that file for precise conditions of the license).
```

> THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
> (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
> OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

> - [FLINT](http://flintlib.org/)

Author: ```suggestion
```

> THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
> (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
> OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

> - [GMP](https://gmplib.org/manual/Copying.html#Copying)
> - [MPFR](https://www.mpfr.org/mpfr-current/mpfr.html#Copying)
> - [MPIR](http://mpir.org/)

Author: ```suggestion
```
